src/axom/sidre/docs/sphinx/data_vs_metadata.rst

```diff
@@ -239,44 +236,54 @@ The output of the code is::
        	Base address of array in A group: 0xc595b8
        	Base address of array in A group: 0xc595b8
 
-  After destroyGroup(A_grp) call:
-       	Num buffers in datastore: 1
-        Is buffer allocated? 1 
-        Num views attached to buffer: 2
+  After destroyGroup(A) call:
        	Num views in group B: 1
        	aview2 in B group has values:	1   3   5   7   9
+       	Num buffers in datastore: 1
+        Is buffer allocated? 1 
+        Num views attached to buffer: 1
 
-  After destroyGroup(B_grp) call:
+  After destroyGroup(B) call:
         Num buffers in datastore: 1
         Is buffer allocated? 1
-        Num views attached to buffer: 2
+        Num views attached to buffer: 0
   
+The last operation in this example is intended to emphasize the explicit 
+nature of Sidre methods. In particular, when a group is destroyed, its views 
+are also destroyed, but their *data may remain intact*. As shown earlier, 
+there are methods provided to destroy views and their deallocate their 
+associated data. Some of this may seem unexpected, but these were early
+design choices for Sidre to yield maximum flexibility in defining and 
+manipulating data hierarchies while keeping internal bookkeeping implementations
+reasonably simple. The example following this one continues on this point.
+
 -------------------------------------
 Example 4: More Basic Mechanics
 -------------------------------------
 
 The last example should help to make clear the point made at the beginning of
-this section and at the end of example three about all Sidre operations being 
-simple and explicit.
+this section and at the end of previous example about Sidre operations 
+being explicit with respect to whether data associated with a group or view
+is destroyed.
 
-We start with a fresh datastore with one group "A" in the root group. We
+We start with a fresh datastore and create one group "A" in the root group. We
 create a view "aview" in the group and allocate it. The output of the 
-code below shows that everything is working as expected. We know what to expect
-because this part is the same as the first part of example one.
+code below shows that everything is working as expected. 
 
 We destroy the view and we see that the view is gone from the group, but
 that the buffer is still in the datastore and allocated. This is so because
-we did not deallocate the view or destroy the buffer.
+we did not explicitly destroy or deallocate the data.
 
-Next, we recreate the view, attach the buffer to it and describe it as before. 
-This restores the state of everything before we destroyed the view.
+Next, we create the view again and attach the buffer to it. Then,i we apply the
```

Comment:
'i we' -> 'we'

Reply:
thanks. that's what happens when one uses via and presses the 'i' key twice instead of once.... 😄